### DISCUSSION TITLE
tutorial feedback?

### DISCUSSION THREAD

**Discussion by preinagel** [bonsai-rx/bonsai#1516] [Ideas]
The Getting Started tutorials are fantastic, thank you for making them. Do you want user feedback (e.g. instructions that don't work in current version or are missing key information; broken links)? If so where is the right place to send/post that?

**Comment by bruno-f-cruz** [comment]
Heya! Thanks :)

Sure, we are always looking for help from the community. The best way to collaborate is to use the "Improve this Doc" that appears on each doc's page. This link will prompt you to make a Pull Request to the docs. This way we can keep track of all changes and also acknowledge people that make the contribution :)

Looking forward for your contribution,
Bruno

**Reply by preinagel** [reply]
ah - just realized that the tutorial I'm looking at is at bonvision.github.io, not bonsai-rx 
but good to know how to contribute on this platform.
